### DISCUSSION TITLE
adding dialogue in kade engine 1.6.2

### DISCUSSION THREAD

**Discussion by Alt-Reality** [KadeArchive/Kade-Engine#1816] [Q&A] [answered]
im trying to add dialogue for a song in a mod im making, but before doing that, i wanted to shift all my assets from 1.6 to 1.6.2 since i wanted the linux modcharts to work. while trying to add the dialogue, i noticed that one thing in playstate.hx was different compared to the older versions and the guides i was watching to make the mod. if you dont understand, heres a picture of what im talking about.
![Inked129653978-60ca7051-e139-4fe1-a2d6-4b4fb7121047_LI](https://user-images.githubusercontent.com/88809517/129654283-24ec2e63-446c-44be-a929-2dcb3d9042aa.jpg)

i really want to make this mod on 1.6.2 instead of an older version, but i dont understand how to add dialogue in this version.
someone please tell me a full guide on how to add dialogue in the game as the guide i was using is really old and used kade engine 1.4.2.

**Comment by isakube** [comment]
unrelated but i like your writing

if you actually looked you would notice we optimized the dialogue code to only check for week 6 songs

line 460

**Reply by Alt-Reality** [reply]
sorry but im still confused. should i just add the song name into the list of week 6 songs on line 459, or do i do something else?

**Comment by RokyDev** [comment, marked as answer]
"case 'senpai' | 'roses' | 'thorns | your-song-name':"

**Reply by Alt-Reality** [reply]
oh ok. thanks for the help. But after I do that, is there anything else i need to do?

**Reply by RokyDev** [reply]
https://gamebanana.com/tuts/13798 and click on 1.8
